src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx

```diff
@@ -372,7 +396,7 @@ const AddStockUserRoleScope: React.FC<AddStockUserRoleScopeProps> = ({
               <Checkbox
                 onChange={onEnabledChanged}
                 checked={formModel?.enabled}
-                labelText={t(`Enabled ?`)}
+                labelText={t("enabled", "Enabled ?")}
```

Comment:
I think you need to update the translations

Reply:
@hadijahkyampeire the translations already exits, it seems the key was drop and that is what I have restored and it works fine.

Reply:
okay then, let's merge it